test/lib/BedrockTester.h

```diff
@@ -120,5 +125,9 @@ class BedrockTester {
     uint16_t _nodePort;
     uint16_t _controlPort;
     uint16_t _commandPortPrivate;
+
+    bool _enforceCommandOrder = false;
+    atomic<uint64_t> _commitCountBase = 0;
+    atomic<uint64_t>& _commitCount;
```

Comment:
just trying to confirm what I think is happening here:

we don't need to instantiate this with `0` even thought it's a reference which _could_ possibly point to some random spot in memory b/c when creating a BedrockTester object here: https://github.com/Expensify/Bedrock/blob/8c7a966f80ed110316accf4d900c9f29e2fc6ce5/test/lib/BedrockTester.cpp#L49

we'll always set this value?

Reply:
It's a reference, so we cant' initialize it with `0`, we need to initialize it with an `atomic<uint64_t>`. Also, because it's a reference, it *can't* be uninitialized. The quoted line either initializes it to whatever's passed in as `alternateCounter` or, to `_commitCountBase` if nothing is passed in.

Reply:
ah yeah okay i had it slightly mixed up. I think I caused some weird behavior one time where I did something like this with a reference but then initialized it to `0` but 👍